noir-projects/aztec-nr/authwit/src/entrypoint/fee.nr

```diff
@@ -72,5 +75,8 @@ impl FeePayload {
                 }
             }
         }
+        if self.is_fee_payer {
+            context.set_as_fee_payer();
+        }
```

Comment:
IIUC the safety comes from the fact that the fee payload as a whole has to pass an authwit check?

Reply:
Yep!

Reply:
Ah damn, you were right, the fee payload as whole is skipped in the authwit check, only the function calls are picked. Gonna fix that now.

Reply:
Wait, doens't it get picked up here? https://github.com/AztecProtocol/aztec-packages/blob/78f13d9e0070502a980003c7ded3738fd353dd5a/noir-projects/aztec-nr/authwit/src/account.nr#L39-L40

Reply:
It does! I had got mixed up, that's why I deleted the comment that it didn't, evidently at the same time as you posted that :-P